translations/strings.yaml

```diff
@@ -64,21 +64,6 @@ feedbackForm:
   reviewBody: If you enjoy using Mozilla VPN, would you review it? Thanks for your support!
   leaveReviewButton: Leave a Review
 
-settingsDev:
-  title: Developer Options
-  showOptionTitle: Show Developer Options
-  showOptionSubtitle: Show or hide the developer options menu
-  useStagingTitle: Use Staging Servers
-  useStagingSubtitle: Staging services are used to test features before they go live to production
-  restartRequired: VPN must be restarted to take effect
-  featureListTitle: Feature list
-  editableFeatureList:
-    value: Editable feature list
```

Comment:
I don't think "editable" makes a lot of sense here. Maybe "Features that can be toggled" or something similar?